changelog/1.24.0.md

```diff
@@ -3,6 +3,9 @@ title: "1.24.0"
 description: "Released on 10/20/2021"
 ---
 
+> Beginning with Coder v1.24.0, the `cdr/coder-cli` repo on GitHub is
+> closed-source.
```

Comment:
Beginning with Coder v1.24.0, the Coder CLI source code is closed-source. The open source repo will continue to exist in it's current form permanently. We will be publishing a new Go SDK to a new repository soon.